languages/english.py

```diff
@@ -310,12 +344,16 @@ def p_print_expr(p):
 def p_value_expr(p):
     '''
     value_expr : value_expr operator value_expr
+                | value_expr operand
                 | operand
     '''
     if len(p) == 4:
         var1 = p[1]
         var2 = p[3]
         python_code = convert_english_pseudocode_to_python(p[2], variable1 = var1, variable2 = var2)
+    elif len(p) == 3:
+        python_code = p[1] + " " + p[2]
```

Comment:
Would these be of the same type to concatenate? Will 'operand' not resolve to numbers etc.?

Reply:
Not necessarily but I added this for "not obstacle ahead" because obstacle ahead was already in operand rule

Reply:
I'm wondering if it'll cause anything unexpected, or allow syntax we don't want to support?

Reply:
Haan now I have changed it to operand value_expr...I'll actually do one thing, I'll write a separate grammar rule for unary operator?